cloudbuild.yaml

```diff
@@ -42,12 +42,6 @@ steps:
     apt-get update
     apt-get install apt-transport-https ca-certificates -y
```

Comment:
we can remove those as well, as we're not fetching bazel anymore.

Reply:
I'm not sure if we can do that but I'll try that out. Those lines were added on OpenROAD bump 2 years ago: https://github.com/hdl/bazel_rules_hdl/pull/56

Reply:
ah interesting I though they were added because of the `wget`, I think we should be able to assume that the official bazel image has everything needed to resolve `http_archive` from the `WORKSPACE`.